### PR TITLE
Core 3139 package structure

### PR DESCRIPTION
**Pull request by tzulberti-jampp**
* [x] Updated [CHANGELOG file](https://github.com/jampp/migratron/blob/master/CHANGELOG.rst)
* [x] Created test for the code changes
* [x] If required, [bumped the version](https://semver.org/)
* [x] If required, updated the requirements.txt file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
 
 import os
+
 from setuptools import setup, find_packages
 
-current_dir = os.path.dirname(os.path.abspath(__file__))
 
-with open(os.path.join(current_dir, "migratron", "VERSION"), "r") as vf:
-    version = vf.read().strip()
+cmd_class = {}
```
Comment (mauguignard) [inline setup.py:9]: I think this is unused.

File: setup.py [hunk 2]
```diff
@@ -1,42 +1,43 @@
...
-    with open(os.path.join(current_dir, filename)) as requirements_file:
-        requirements = requirements_file.readlines()
-        # remove whitespaces
-        requirements = [line.strip().replace(" ", "") for line in requirements]
-        # remove all the requirements that are comments
-        requirements = [line for line in requirements if not line.startswith("#")]
-        # remove empty lines
-        requirements = list(filter(None, requirements))
-        return requirements
+with open(os.path.join(setup_dir, "migratron", "VERSION")) as vf:
```
Comment (mauguignard) [inline setup.py:17]: nit: To be consistent with the previous statements, I would use `version_file` instead.

File: setup.py [hunk 3]
```diff
@@ -1,42 +1,43 @@
...
+    # but without the fixed version. For more info check
+    # https://packaging.python.org/discussions/install-requires-vs-requirements/
+    install_requires=[
+        "pygments",
+        "psycopg2",
+    ],
+    license="BSD 3-Clause",
+    long_description=readme + "\n\n" + history,
+    long_description_content_type="text/x-rst",
+    package_data={"migratron": ["VERSION"],},
```
Comment (mauguignard) [inline setup.py:37]:
```suggestion
    package_data={"migratron": ["VERSION"]},
```

Comment (tzulberti-jampp) [reply setup.py:37]: That was added by black

**Pull request merged**